### PR TITLE
tinkerbell: Move to fedora based image

### PR DESCRIPTION
**Pull request by surajssd**
`cfssl` package for alpine has been removed. Hence the build fails. This
commit moves to a fedora based image.

### CODE REVIEW

**Comment by surajssd** [comment]
Created a PR upstream: https://github.com/tinkerbell/sandbox/pull/86

**Review by ipochi — APPROVED** [review]
lgtm

File: assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/entrypoint.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```
Comment (invidian) [inline assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/entrypoint.sh:1]: Why this?

File: assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/gencerts.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
```
Comment (invidian) [inline assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/gencerts.sh:1]: Why is this change needed?

**Review by invidian — COMMENTED** [review]
`cfssl` is the only binary used, so so long versions are matching and this is reported upstream, I think it should be fine.

However, I'd be very cautious making any changes to vendored code in this repository, as if this continuous, it makes code very hard to maintain.

**Review by invidian — APPROVED** [review]

**Review by iaguis — APPROVED** [review]

File: assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/entrypoint.sh (hunk 1 above)
Comment (surajssd) [reply assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/entrypoint.sh]: This is removed.

File: assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/gencerts.sh (hunk 2 above)
Comment (surajssd) [reply assets/terraform-modules/tinkerbell-sandbox/assets/deploy/tls/gencerts.sh]: I think I added it because we moved away from alpine to fedora, and bash being readily available in fedora, I thought might as well switch to it. But it turns out `/bin/sh` is a sym link to `/bin/bash`. So I guess it is fine.
